CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Unreleased
+- On Wayland, windows will use server-side decorations when appropriate.
```

Comment:
I'd rather word it "when available", as otherwise the notion of what "appropriate" means is unclear.